test/functional/wallet_migration.py

```diff
@@ -538,10 +538,16 @@ def test_default_wallet(self):
         assert_equal(info["descriptors"], True)
         assert_equal(info["format"], "sqlite")
 
+        walletdir_list = wallet.listwalletdir()
+
         # Check backup existence and its non-empty wallet filename
-        backup_path = self.nodes[0].wallets_path / f'default_wallet_{curr_time}.legacy.bak'
+        backup_filename = f"default_wallet_{curr_time}.legacy.bak"
+        backup_path = self.nodes[0].wallets_path / backup_filename
         assert backup_path.exists()
         assert_equal(str(backup_path), res['backup_path'])
+        assert {"name": backup_filename} not in walletdir_list["wallets"]
+
+        wallet.setmocktime(0)
```

Comment:
this line does not seems to be related to a700b303fcba7915f.

Reply:
Removed